js/csv.js

```diff
@@ -35,54 +35,40 @@ var csv = (function () {
-                                <tr>
-                                    <th scope="col"></th>
-                                    <th scope="col" i18n="csv.srs.lon">X (longitude)</th>
-                                    <th scope="col" i18n="csv.srs.lat">Y (latitude)</th>
-                                    <th scope="col" i18n="csv.srs.projection">Projection (SRS)</th>
-                                </tr>
-                                </thead>
-                                <tbody>
-                                <tr>
-                                    <th scope="row" i18n="csv.srs.select">Sélectionner</th>
-                                    <td>
-                                        <select id="x-select" class="form-control"></select>
-                                    </td>
-                                    <td>
-                                        <select id="y-select" class="form-control"></select>
-                                    </td>
-                                    <td>
-                                        <select id="srs-select" class="form-control"></select>
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <th scope="row" i18n="csv.srs.data">Extrait des données</th>
-                                    <td id="x-data"></td>
-                                    <td id="y-data"></td>
-                                    <td></td>
-                                </tr>
-                                </tbody>
-                            </table>
-                            <div>
-                                <button id="choose-fields" type="button" data-layerid="" class="geocode btn btn-primary pull-right" i18n="csv.button.back">Retour</button>
-                            </div>
-                        </div>
+                    <div class="panel-body">
+                        <table id="table-csv" class="table">
+                            <thead>
+                            <tr>
+                                <th scope="col"></th>
+                                <th scope="col" i18n="csv.srs.lon">X (longitude)</th>
+                                <th scope="col"></th>
+                                <th scope="col" i18n="csv.srs.lat">Y (latitude)</th>
+                                <th scope="col" i18n="csv.srs.projection">Projection (SRS)</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <th scope="row" i18n="csv.srs.select">Sélectionner</th>
+                                <td>
+                                    <select id="x-select" class="form-control"></select>
+                                </td>
+                                <td><button id="btn-xy" class="btn glyphicon glyphicon-refresh"></button></td>
```

Comment:
Suggest to specify color like that btn-primary or btn-warning and size you like it :
`class="btn btn-sm btn-warning glyphicon glyphicon-refresh`

or 

`class="btn btn-sm btn-primary glyphicon glyphicon-refresh`